### DISCUSSION TITLE
`about:config` in qutebrowser

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [qutebrowser/qutebrowser#7893] [Q&A]
I was browsing my university's web site today and saw somewhere that they recommended changing this values in the browser by going  to `about:config`
Is there something appropriate to these in qutebrowser ?  thank you
```
network.dns.disableIPv6 
network.http.pipelining 
network.http.proxy.pipelining 
network.http.pipelining.maxrequests
```

**Comment by The-Compiler** [comment]
`about:config` is a Firefox thing. You'd need to find out how to set those things in Chromium. It might not be possible at all, though. If it's possible via something like [CLI switches](https://peter.sh/experiments/chromium-command-line-switches/), you might be able to add them to the `qt.args` setting in qutebrowser.
